test/Microsoft.ML.TestFrameworkCommon/Datasets.cs

```diff
@@ -470,6 +470,13 @@ public static class TestDatasets
             testFilename = @"iris-test"
         };
 
+        public static TestDataset irisDbSQLite = new TestDataset()
```

Comment:
Can you keep the table names the same as above and not need this separate entry?

Reply:
Hi Harish,
In SQLite table names including the '-' characters are not valid. The underscore '_' is the only valid separator character afaik. I understand I am making an addition here, but this change is necessary to avoid invalid SQLite table names. This is also for a good reason, as I found I had trouble with SQLite queries with table names containing the '-' character.

Reply:
Can you please add a comment to that effect in code?

---
In reply to: [419010912](https://github.com/dotnet/machinelearning/pull/5080#discussion_r419010912) [](ancestors = 419010912)

Reply:
Sure, I'll add an explanation for the existence of `TestDatasets.irisDbSQLite`. To keep commits to the minimum amount, I will add all necessary changes once the up-to-date NuGet package for the `machinelearning-testdata` repo is available, where the necessary `iris.sqlite` is in its `TestDatasets` directory.